command/forceleave/forceleave.go

```diff
@@ -1,31 +1,48 @@
-package command
+package forceleave
 
 import (
+	"flag"
 	"fmt"
-	"strings"
+
+	"github.com/hashicorp/consul/command/flags"
+	"github.com/mitchellh/cli"
 )
 
-// ForceLeaveCommand is a Command implementation that tells a running Consul
-// to force a member to enter the "left" state.
-type ForceLeaveCommand struct {
-	BaseCommand
+func New(ui cli.Ui) *cmd {
+	c := &cmd{UI: ui}
+	c.init()
+	return c
+}
+
+type cmd struct {
+	UI    cli.Ui
+	flags *flag.FlagSet
+	http  *flags.HTTPFlags
+	help  string
+}
+
+func (c *cmd) init() {
+	c.flags = flag.NewFlagSet("", flag.ContinueOnError)
+	c.http = &flags.HTTPFlags{}
+	flags.Merge(c.flags, c.http.ClientFlags())
+	flags.Merge(c.flags, c.http.ServerFlags())
+	c.help = flags.Usage(help, c.flags, c.http.ClientFlags(), c.http.ServerFlags())
```

Comment:
Why do we have to repeat `c.http.ClientFlags(), c.http.ServerFlags()` in `Usage()` if they are already merged into `c.flags`?

Reply:
I can simplify that.

Reply:
I do like the explicit merge approach vs. the bitfields. I'm wondering if we can search for the ClientFlags and ServerFlags when we render the usage and group them, and filter them out otherwise, since we know what they are.

Reply:
You're right. With the current code this is no longer necessary. I've refactored the commands so that they extract the HTTP API flags automatically.